### ISSUE TITLE
Definition of prof:isProfileOf

### ISSUE THREAD

**Issue by kcoyle** [w3c/dxwg#1068]
The current definition of `prof:isProfileOf` is:

>The subject of 'is profile of' defines constraints on the object which playes the role of a base specification

As "specification" has been defined, not all specifications are profiles. The "axiomatization" of isProfileOf (#844) implies that the object of `isProfileOf` is a profile. 

I see two things that need to be clarified:
1) what is the relationship of a profile to the object of `isProfileOf`? It may constrain but it also may expand on, so defining this as constraints does not convey that.
2) what is the object of `isProfileOf`? This has been discussed elsewhere (e.g. #507) but I cannot find a conclusion. If the object is not a profile itself, then the axiomatization in #844 needs to reflect that.

**Comment by nicholascar** [comment]
> The "axiomatization" of isProfileOf (#844) implies that the object of isProfileOf is a profile.

The proposed axiom is:
```
dct:conformsTo owl:PropertyAxiom ( dct:conformsTo prof:isProfileOf )
```

This doesn't make any claim about the range value of `prof:isProfileOf`, only that if X conforms to Y and Y isProfileOf Z, then X conformsTo Z.

Here Y would have to be a profile as it's indicated as being as isProfileOf Z. Z could be a specification that doesn't profile anything, so the conformance chain ends there.

**Comment by rob-metalinkage** [comment]
The questions are already addressed:

prof:isProfileOf rdfs:range dct:Standard

doesnt need any further decisions, and answers the questions posed.

The axiom makes the relationship clear within the formalism of OWL semantics, and its been stated in text already. This was relaxed from rdfs:range prof:Profile even though that would be correct, because it seems too hard to understand that any dct:Standard may be a profile of something else in an "Open World" interpretation, and of itself under principles of reflexivity.  Possibly we should explicitly declare profileOf as a reflexive property (w3.org/TR/owl2-syntax/#Reflexive_Object_Properties).

(To understand Profiles are analogous to rdfs:Classes - they describe a set of individual instances that conform to them - where the instances are datasets, not sets of RDF statements about a subject - and hence Profiles naturally exhibit transitivity and reflexivity. We avoided doing this just in the most formal and technical way because it should be easier for _users_ of the spec to deal with concepts of Profiles and data without being versed in computer science - but likewise where we need to be specific about behaviour then the _authors_ need to understand the computer science to design the specification so it says just what it needs to say and doesnt introduce logical errors)

The proposal for this issue is that including the axiom is necessary because text is too easily misinterpreted.  There is simply no intention to try to create a model for all possible relationships between specifications where no assertions about conformance can be made. 

The misunderstanding here is about what a statement about conformance actually implies ;

"expand on" is just adding constraints about the semantics and cardinality of additional terms - it doesnt change any requirements about conformance to the profiled specification.  Just because languages dont use the generic term "constraints" doenst mean it doesnt apply - OWL semantics are constraints, sko:Concepts constrain interpretation of terms etc.

**Comment by aisaac** [comment]
+1 for the range as it is.

Re. "expand" I guess this related to what we discuss (also wrt "combines" in the definition) in #507. Maybe it's better to have the discussion there?

Note that I had created #1061 as an attempt to have discussions on the definition (like this one - there would be no need to be worried about the wording with "constraints" if other words from the agreed WG definition were there!) based on a text that is a bit fresher. But I've failed...

**Comment by kcoyle** [comment]
Ah, sorry @aisaac I missed your issue on definitions. I'll add any suggestions I have over there and we can close them both together when we finalize the definitions.

**Comment by kcoyle** [comment]
To be closed when #1061 reaches closure.

**Comment by nicholascar** [comment]
Closing since the definition, with changes from #1061, has been updated and published in the NOTE.
